packages/contracts-bedrock/contracts/test/L1StandardBridge.t.sol

```diff
@@ -10,14 +10,17 @@ import { AddressAliasHelper } from "../vendor/AddressAliasHelper.sol";
 import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 import { stdStorage, StdStorage } from "forge-std/Test.sol";
 
-contract L1StandardBridge_Test is Bridge_Initializer {
-    using stdStorage for StdStorage;
-
-    function setUp() public override {
-        super.setUp();
+contract L1StandardBridge_Getter_Test is Bridge_Initializer {
```

Comment:
Does `--match` account for contract names or just function names? This format might make it annoying to test individual functions. We could work around this by having the test names still include the contract name.

Reply:
Just function names for `--match`. Though there are a bunch of variants too that you can use:

```
--match-test <REGEX>                   Only run test functions matching the specified regex pattern [aliases: mt]
--match-contract <REGEX>               Only run tests in contracts matching the         --match-path <GLOB>                    Only run tests in source files matching the specified glob pattern [aliases: mp]
```

And you can combine them, ie. this runs all L1 Bridge tests with the word 'deposit' in the name: `ft --mc L1StandardBridge --mt deposit`. 

Given those options, I think the current style is cleaner.

Reply:
Using `--match` is deprecated, shoudl use `--match-test` and `--match-contract`